keps/sig-node/3673-kubelet-parallel-image-pull-limit/README.md

```diff
@@ -646,13 +691,11 @@ and operation of this feature.
 Recall that end users cannot usually observe component logs or access metrics.
 -->
 
-- [ ] Events
-  - Event Reason:
-- [ ] API .status
-  - Condition name:
-  - Other field:
-- [ ] Other (treat as last resort)
-  - Details:
+- [X] Events
+  - Event Reason: Pulling
+
+Assuming `MaxParallelImagePulls` is set to _X_, an operator can look at the container runtime log, and see _X_ PullImageRequests sent to container runtime at the same time.
+If the container images are of similar sizes, an operator can see k8s event and see _X_ images finish pulling at roughly the same time.
```

Comment:
Not sure if this will be true if the images are being pulled off different registries or from different servers with different loads.

Reply:
yeah this will only be true when the pulling takes roughly same time for all the images. If different pulls take different amount of time, then an operator will not be able to deduce the parallel behavior.